custom_components/clientraw/sensor.py

```diff
@@ -295,16 +327,16 @@ def try_again(err: str):
 
             elif dev.type == 'forecast':
                 val = int(self.data[15])
-                arr = ["sunny", "clearnight", "cloudy", "cloudy2",
-                       "night cloudy", "dry", "fog", "haze", "heavyrain",
-                       "mainlyfine", "mist", "night fog", "night heavyrain",
+                arr = ["sunny", "clear night", "cloudy", "cloudy2",
+                       "night cloudy", "dry", "fog", "haze", "heavy rain",
+                       "mainly fine", "mist", "night fog", "night heavy rain",
                        "night overcast", "night rain", "night showers",
-                       "night snow", "night", "thunder", "overcast",
-                       "partlycloudy", "rain", "rain2", "showers2", "sleet",
-                       "sleetshowers", "snow", "snowmelt", "snowshowers2",
-                       "sunny", "thundershowers", "thundershowers2",
-                       "thunderstorms", "tornado", "windy", "stopped",
-                       "rainning", "wind + rain"]
+                       "night snow", "night thunder", "overcast",
+                       "partly cloudy", "rain", "rain2", "showers", "sleet",
+                       "sleet showers", "snow", "snow melt", "snow showers2",
+                       "sunny", "thunder showers", "thunder showers2",
+                       "thunder storms", "tornado", "windy",
+                       "stopped raining"]
```

Comment:
Well spotted  `stoped raining`
I would like to keep the `wind + rain` there is apparently also `sunrise` and `sunset`
https://git.dev.hsdn.org/pub/wdparser/src/master/src/WeatherDisplay.php#L1203

Reply:
Sure - I hadn't realised that - the parser I use doesn't have those extras so I will add

Reply:
Sorry for delay - ok those added